lib/utils/job-utils/ipmitool.js

```diff
@@ -163,16 +162,44 @@ function ipmitoolFactory(Promise) {
     };
 
     /**
-     * Returns a promise with the results or errors of invoking sel list -c
+     *  Returns a promise with the results or errors of invoking sel list -c
      *
      * @param host
      * @param user
      * @param password
      * @param count
      */
-    Ipmitool.prototype.sel = function(host, user, password, count) {
+    Ipmitool.prototype.sel = function(host, user, password, count) {
         return this.runCommand(host, user, password, "-c sel list last " + count);
     };
+    
+    /**
+     * Returns a promise with the results or errors of invoking sel get <entry> ...<entry>
+     *
+     * @param host
+     * @param user
+     * @param password
+     * @param Sel Entries
+     */
+    Ipmitool.prototype.selEntry = function(host, user, password, stringEntries) {
+        return this.runCommand(host, user, password, "sel get " + stringEntries);
+    };
+
+    /**
+     * Returns a promise with the results or errors of invoking sel raw
+     *
+     * @param host
+     * @param user
+     * @param password
+     * @param Sel Entries
+     */
+    Ipmitool.prototype.readRawSel = function(host, user, password, selEntry) {
+        var tmp = selEntry.split("") ;
+        var command = "raw 0x0a 0x43 0x00 0x00 " + 
```

Comment:
Please add some comment for these constants, otherwise other developer will not be aware of what "0x0a" and "0x43" means.